app/forms/provider_interface/offer_wizard.rb

```diff
@@ -0,0 +1,58 @@
+module ProviderInterface
+  class OfferWizard
+    include ActiveModel::Model
+
+    STEPS = { make_offer: %i[select_option conditions check] }.freeze
```

Comment:
How come this appears to be dependent on the decision?

Reply:
Because the selected decision controls which steps are displayed in the original flow.